src/states/nebraska/main.py

```diff
@@ -56,17 +58,17 @@ def parse_county(county):
     'locale': county,
     'official': result['Name'],
     'emails': [result['Email Address']],
-    'faxes': [result['Fax Number']],
-    'phones': [result['Phone Number']],
+    'faxes': re_phone_fax.findall(result['Fax Number']),
```

Comment:
I'm afraid that we may mis some slightly malformed numbers. Could we be more generous on the intake and then write a more robust substitution in re_to_e164?